examples/logistic/common.py

```diff
@@ -0,0 +1,159 @@
+import tensorflow as tf
+import tf_encrypted as tfe
+
+
+class LogisticRegression:
+
+    def __init__(self, num_features):
+        self.w = tfe.define_private_variable(tf.random_uniform([num_features, 1], -0.01, 0.01))
+        self.w_masked = tfe.mask(self.w)
+        self.b = tfe.define_private_variable(tf.zeros([1]))
+        self.b_masked = tfe.mask(self.b)
+
+    @property
+    def weights(self):
+        return self.w, self.b
+
+    def forward(self, x):
+        with tf.name_scope("forward"):
+            out = tfe.matmul(x, self.w_masked) + self.b_masked
```

Comment:
Would it be worth using tfe.layers.Dense here?

Reply:
(although perhaps best to wait until the layer implementations have been cleaned up)

Reply:
definitely, even thought the model could eventually move into TFE. suggest we do that as part of high-level API update!